aws/outputs.tf

```diff
@@ -2,3 +2,8 @@ output "blockscout_url" {
   description = "DNS name of frontend"
   value       = module.alb.lb_dns_name
 }
+
+output "xchain_url" {
```

Comment:
what's this for?

Reply:
for cloudflare

Reply:
I mean more generally, why do we need a new url?

Reply:
oh for the xchain api server

Reply:
hmm is there a reason it needs to be a public api? We should keep it internal to the VPC if we can

Reply:
yes because the UI dynamically calls it from the users browser so it has to be exposed. It's pretty similar to any react app, where the website is static and any data is fetched from exposed apis of the backend. Blockscout also exposes apis it's just doesn't need a separate URL because it's hosted on the same server as the UI.